embedded-influx-junit5/build.gradle

```diff
@@ -23,4 +23,10 @@ dependencies {
     testImplementation libs.influxClient
     testImplementation libs.junit5
     testImplementation libs.kotlinTest
-}
\ No newline at end of file
+    
+}
+
+test {
+    useJUnitPlatform {
```

Comment:
Also unclear on what this does; don't tests run correctly already?

Reply:
The JUnit 5 tests won't run without it.  If you add a `TODO` to a test in the JUnit5 module, then run `./gradlew clean test`, it won't blow up the build.

Reply:
Huh, TIL.